bin/hermes

```diff
@@ -32,7 +33,15 @@ def request_get(path):
     Returns:
         the http response
     """
-    response = requests.get(settings.hermes_server + path)
+    retries = 0
+    clear_response = False
+    while not clear_response and retries < 5:
```

Comment:
5 is the international standard for retries.  But configurable makes sense.  If this solved my issue I'd expand it to support configuration and a command line param to override.